src/CloudNimble.PackageReferenceUpgrader/NuGetUpgraderPackage.cs

```diff
@@ -175,16 +175,19 @@ private void UpgradePackagesConfig()
                             new XAttribute("Version", row.Attribute("version").Value)));
 
                         //RWM: Remove the old Standard Reference.
-                        oldReferences.Where(c => c.Attribute("Include").Value.Split(new Char[] { ',' })[0].ToLower() == row.Attribute("id").Value.ToLower()).ToList()
+                        if (oldReferences != null) oldReferences.Where(c => c.Attribute("Include") != null).Where(c => c.Attribute("Include").Value.Split(new Char[] { ',' })[0].ToLower() == row.Attribute("id").Value.ToLower()).ToList()
```

Comment:
Were there exceptions happening here? Because of the way LINQ works, I assumed that `oldReferences` would be a blank collection 100% of the time.

Two observations:
1) There is only one situation where you should EVER do an if statement without braces: when you're checking parameters to short-circuit a function. (`if (someParameter == null) return;`), If you go outside of that rule, you risk creating the exact same issue that affected OpenSSL.
2) This is great, and I really appreciate it. Wouldn't this be easier by just using a [null conditional operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-conditional-operators) (`oldReferences?.Where(...)`)?